packages/components/tests/e2e/specs/appmap/viewFilter.spec.js

```diff
@@ -125,10 +125,10 @@ context('AppMap view filter', () => {
       cy.get('.trace .trace-node').should('have.length', 5);
     });
 
-    it('hides unlabeled code', () => {
+    it.only('hides unlabeled code', () => {
```

Comment:
Hi @moxw can you please remove the `.only` call?

Reply:
Removed.

Reply:
I still see it

Reply:
Apologies, was pushing it from wrong directory, should be fixed now.